packages/yourloops/lib/patient/models/alarms.model.ts

```diff
@@ -33,3 +34,42 @@ export interface Alarms {
   nonDataTransmissionRate: number
   nonDataTransmissionActive: boolean
 }
+
+export interface Thresholds {
+  minHighBg: number
+  maxHighBg: number
+  minVeryLowBg: number
+  maxVeryLowBg: number
+  minLowBg: number
+  maxLowBg: number
+}
+
+export interface BgValues {
+  bgUnitDefault?: UnitsType
```

Comment:
I see that in your last commit, you made most of these values nullable, can you tell me why?